tests/integration/activities.intTest.ts

```diff
@@ -68,34 +68,93 @@ describe("PUT /activities/:id/end", () => {
     context("when a non-existing activity is ended", () => {
         it("should respond with HTTP 201", () => {
             return request.put(`/activities/bad_id/end`)
-            .send({})
-            .expect("access-control-allow-origin", "*")
-            .expect("access-control-allow-credentials", "true")
-            .expect(404);
+                .send({})
+                .expect("access-control-allow-origin", "*")
+                .expect("access-control-allow-credentials", "true")
+                .expect(404);
         });
     });
 
     context("when an existing activity is ended", () => {
         it("should respond with HTTP 204", () => {
             return request.put(`/activities/${postedActivity.id}/end`)
-            .expect("access-control-allow-origin", "*")
-            .expect("access-control-allow-credentials", "true")
-            .expect(204);
+                .expect("access-control-allow-origin", "*")
+                .expect("access-control-allow-credentials", "true")
+                .expect(204);
         });
     });
 
     context("when an already ended activity is ended", () => {
         it("should respond with HTTP 403", () => {
             return request.put(`/activities/${postedActivity.id}/end`)
-            .expect("access-control-allow-origin", "*")
-            .expect("access-control-allow-credentials", "true")
-            .expect(403);
+                .expect("access-control-allow-origin", "*")
+                .expect("access-control-allow-credentials", "true")
+                .expect(403);
         });
     });
 
     after((done: Done) => {
         activityService.dbClient.delete(postedActivity)
             .then(() => done());
     });
+});
 
+describe("GET /activities/details", () => {
```

Comment:
Can you please split the integration tests in multiple files? I would suggest the split to be done by endpoints.